ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -64,6 +64,9 @@ if (typeof window.FileInputWidget === 'undefined') {
         }
 
         #attachEventHandlers(widget, model) {
+            widget.addEventListener('click', (e)=> {
```

Comment:
Please add test cases

Reply:
Test cases are added and the code is sync'd with @rajatofficial's drag and drop code update